### PR TITLE
Improve logging for Clinica

### LINKED ISSUE #259: Clean up Clinica logs
**Is your feature request related to a problem? Please describe.**
Currently, most of the pipeline outputs are printed on screen which results in very verbose and counterproductive output.

**Describe the solution you'd like**
Create logs to capture some of the current outputs and hierarchize the  information according to the different logging levels (c.f https://docs.python.org/3/howto/logging.html)

### PR DESCRIPTION
**Pull request by ghisvail**
Closes #259

### CODE REVIEW

**Review by mdiazmel — COMMENTED** [review]
Thanks @ghisvail ,

This PR substantially improves the readability of the code. That's great! 

I have some small comments about when and where to use the different levels of verbosity (see inside the review). 

There are some other issues concerning the order of the import statements. See these files:
```
clinica/cmdline.py
clinica/engine/cmdparser.py
clinica/iotools/converters/adni_to_bids/adni_to_bids.py
clinica/iotools/converters/adni_to_bids/adni_utils.py
clinica/pipelines/pet_volume/pet_volume_utils.py
clinica/pipelines/pet_linear/pet_linear_utils.py
```

We use [isort](https://pycqa.github.io/isort/) to solve them automatically. You can install the plugin for you editor, run it manually, or even better install the pre-commit hook in your clone.

File: clinica/iotools/utils/data_handling.py [hunk 1]
```diff
@@ -1089,26 +1073,27 @@ def check_volume_location_in_world_coordinate_system(
...
                     break
                 else:
-                    cprint(Fore.RED + "You must answer yes or no" + Fore.RESET)
+                    cprint(msg="You must answer yes or no", lvl="warning")
         else:
             answer = "yes"
 
         if answer.lower() == "no":
-            cprint(Fore.RED + "Clinica will now exit..." + Fore.RESET)
+            cprint(msg="Clinica will now exit...", lvl="debug")
```
[the remark this answers is not in the document]
Comment (ghisvail) [reply clinica/iotools/utils/data_handling.py:1096]: Feel free to edit it 👍

File: clinica/iotools/utils/data_handling.py [hunk 2]
```diff
@@ -1165,17 +1150,14 @@ def get_world_coordinate_of_center(nii_volume):
...
     assert isinstance(nii_volume, str), "input argument nii_volume must be a str"
     assert isfile(nii_volume), "input argument must be a path to a file"
 
     try:
         orig_nifti = nib.load(nii_volume)
     except nib.filebasedimages.ImageFileError:
-        print(
-            f"{Fore.RED}[Error] {nii_volume}  could not be read by nibabel. Is it a valid NIfTI file ?{Fore.RESET}"
-        )
+        print(f"File {nii_volume} could not be read by nibabel. Is it a valid NIfTI file ?")
```
[the remark this answers is not in the document]
Comment (ghisvail) [reply clinica/iotools/utils/data_handling.py:1160]: Indeed

**Review by omar-rifai — APPROVED** [review]
Sorry for the delay, here are some reviews I had found. Maybe there are some that were redundant with those of Mauricio.

File: clinica/utils/check_dependency.py [hunk 3]
```diff
@@ -140,17 +127,16 @@ def check_freesurfer(version_requirements=None):
     for binary in list_binaries:
         if not is_binary_present(binary):
             raise ClinicaMissingDependencyError(
-                f"{Fore.RED}\n[Error] Clinica could not find FreeSurfer software: "
+                "Clinica could not find FreeSurfer software: "
                 f"the {binary} command is not present in your PATH environment: "
-                f"did you have the line source ${{FREESURFER_HOME}}/SetUpFreeSurfer.sh "
-                f"in your configuration file?{Fore.RESET}"
+                "did you have the line source ${{FREESURFER_HOME}}/SetUpFreeSurfer.sh "
```
[the remark this answers is not in the document]
Comment (ghisvail) [reply clinica/utils/check_dependency.py:132]: missing ``...``

File: clinica/iotools/converters/adni_to_bids/adni_modalities/adni_av45_fbb_pet.py [hunk 4]
```diff
@@ -39,7 +38,7 @@ def convert_adni_av45_fbb_pet(
         "Paths of AV45 and Florbetaben PET images found. Exporting images into BIDS ..."
     )
     paths_to_bids(images, dest_dir, "av45_fbb", mod_to_update=mod_to_update)
-    cprint(f"{Fore.GREEN}AV45 and Florbetaben PET conversion done.{Fore.RESET}")
+    cprint(msg="AV45 and Florbetaben PET conversion done.", lvl="debug")
```
[the remark this answers is not in the document]
Comment (ghisvail) [reply clinica/iotools/converters/adni_to_bids/adni_modalities/adni_av45_fbb_pet.py:41]: After discussion, keeping it as debug.

**Comment by ghisvail** [comment]
@omar-rifai all done 👍

**Comment by mdiazmel** [comment]
LGTM,
@ghisvail , do you mind if we squash the commits into a single one and then we merge?

**Comment by ghisvail** [comment]
@mdiazmel yes please

**Pull request merged**